packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -0,0 +1,90 @@
+---
+id: Upgrade
+title: Major release upgrade guide
+section: get-started
+source: Major-release-upgrade
+---
+
+# PatternFly 5 
+
+Our latest major release introduces new support and functionality to PatternFly, including: 
+
+- React 18 support, 
+- Official dark theme support, 
+- An enhanced system of CSS classes and variables, 
+- And more!
+
+To learn about the most significant changes in this release, read our [release highlights](/get-started/release-higlights.md). A detailed list of all changes can be found in our [major release notes](get-started/upgrade.md#major-release-notes). 
+
+This guide outlines the major steps you should take to upgrade your product's codebase from PatternFly 4 to PatternFly 5. 
+
+## Get help 
+
+If you need support as you upgrade to PatternFly 5, the PatternFly team is here to help. Reach out to us on [Slack](https://join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ) or ask a question on our [GitHub discussion board](https://github.com/orgs/patternfly/discussions). We'll always do our best to answer your questions and connect you with the right people quickly. 
+
+## Upgrade your product's codebase
+
+When you upgrade your product to PatternFly 5, several breaking changes will likely be introduced to your product’s codebase. We are using a suite of Codemods to simplify and streamline the upgrade process. Instead of requiring you to manually identify all errors and issues in your codebase, you can run our Codemods to quickly identify and fix major issues. Keep in mind that some changes will still require manual intervention, but our Codemods can automatically fix a large amount of issues and flag any issues that do require manual work.
+
+To utilize our Codemods, refer to the following instructions. You can also [view the project on GitHub](https://github.com/patternfly/pf-codemods/) for additional details.
+
+###  Run our Codemods
+
+**Note:** Running Codemods after making manual changes will lead to even more manual intervention and cleanup. Because of this, we strongly recommend running Codemods _before_ making any manual changes to your codebase.
+
+To run our Codemods, complete the following steps:
+
+1. Run the following command, adding in the path to your product's source code: 
+
+```{
+npx @patternfly/pf-codemods@latest <path to your source code> --exclude menu-search-rename,alert-remove-titleHeadingLevel,datetimepicker-warn-helperText,formgroup-remove-helpertextProps,formhelpertext-remove-props,alert-remove-titleHeadingLevel
+}
+```
+
+2. Make note of any issues that get flagged.
+
+3. Add the `–fix` flag to the end of your original command and run it again. 
+
+4. Make note of the changes applied to your product's code base.
+
+5. Note any errors that were not automatically fixed (some may simply be warnings) and manually repair issues that are critical.
```

Comment:
Just a tip but you can list all of these directly after one another [like the list above](https://github.com/patternfly/patternfly-org/blob/5052b264e8afa71d2864fe7d71711c2f6ffc6d30/packages/documentation-site/patternfly-docs/content/get-started/upgrade.md?plain=1#L12-L15), and if the list is numbered, you can use `1.` for all of them and markdown will add the numbers in order. That's helpful if you add/remove things, you don't have to update the list item numbers if they all use `1.`

Reply:
oh, this is so helpful to know. Thank you!